allofplos/article_class.py

```diff
@@ -69,13 +69,13 @@ def doi(self):
     def text_viewer(self):
         """Command line application for viewing text to be used with 
         open_in_viewer.
-        
-        Defaults to "open", which opens in whatever the default application is 
+
+        Defaults to "open", which opens in whatever the default application is
```

Comment:
I think the reason @sbassi added that there is that without that space, it is concatenated on the command line and is no longer a valid word.

Reply:
looks fine to me, there is a newline and `print(Article.text_viewer.__doc__)` works as expected.